src/components/dynamic/admin/services/statistics/Statistics.jsx

```diff
@@ -3,6 +3,7 @@ import Title from "@/components/dynamic/admin/Title";
 import { useEffect, useState } from "react";
 import Tabs from "./Tabs";
 import Loading from "@/components/dynamic/Loading";
+import { api } from "@/utils/api";
```

Comment:
remove if not being used?!

Reply:
<img width="331" alt="image" src="https://github.com/acm-ucr/hackathon-website/assets/90938120/926202e5-2cb4-40d1-a799-31220567e7a7">

this is undefined without that include